src/envoy/mixer/envoy.conf.template

```diff
@@ -35,7 +35,62 @@
                 "type": "both",
                 "name": "mixer",
                 "config": {
-                  "mixer_server": "${MIXER_SERVER}"
+                  "mixer_server": "${MIXER_SERVER}",
```

Comment:
If this is environment variable, it should absolutely be killed! ESP in a JSON file.